scripts/irods/test/test_igroupadmin.py

```diff
@@ -72,6 +71,63 @@ def tearDownClass(self):
             self.admin.__exit__()
             admin_session.assert_icommand(['iadmin', 'rmuser', self.admin.username])
 
+    def test_create_user_with_password_in_igroupadmin__6887(self):
+        USERS = { 'alice':42,
+                  'bob':40, 'eve': 3 }
+
+        # TODO -- eventually delete , but using for test r.n.
+        #obj_name = '{}/{}'.format(self.groupadmin.home_collection, 'test__6887__object')
+        #self.groupadmin.assert_icommand(['ichmod', 'write', self.groupadmin.username, obj_name], 'EMPTY' )
+
+        for username,pwlen in USERS.items():
+            try:
+                passwd = (username[0]+'pass_6887_the_quick_brown_fox_jumps_over_the_lazy_dog')[:pwlen]
+                self.groupadmin.assert_icommand(['igroupadmin', 'mkuser', username, passwd], 'EMPTY' )
```

Comment:
Just an FYI, specifying `'EMPTY'` is redundant when there's no output.

So, you can remove that argument if you want. Up to you.

Reply:
Right, I knew that ( I think :) ).

Reply:
done!